conf/broker.conf

```diff
@@ -212,10 +212,7 @@ loadBalancerReportUpdateThresholdPercentage=10
 # maximum interval to update load report
 loadBalancerReportUpdateMaxIntervalMinutes=15
 
-# Path for the script used to retrieve system usage
-loadBalancerHostUsageScriptPath=
-
-# Frequency of sar report to collect
+# Frequency of report to collect
 loadBalancerHostUsageCheckIntervalMinutes=1
```

Comment:
Does it really make sense to have intervals of < 1min? Would we be updating these reports to ZK everytime?